ci/scripts/cleanup-resources.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#
+# (c) Copyright IBM Corp. 2024
+# (c) Copyright Instana Inc.
+#
+
+set -x
+set -e
+set -o pipefail
+
+
+case "${CLUSTER_TYPE}" in
+    gke)
+        echo 'Running on a GKE cluster'
+
+        echo "${GCP_KEY_JSON}" > keyfile.json
+        echo "${CLUSTER_INFO}" > cluster.info
+
+        echo 'Authenticating with gcloud'
+
+        gcloud auth activate-service-account --key-file keyfile.json
+
+        readonly GKE_PROJECT=$(jq -r .project < cluster.info)
+        readonly GKE_CLUSTER_NAME=$(jq -r .name < cluster.info)
+        readonly GKE_ZONE=$(jq -r .zone < cluster.info)
```

Comment:
https://www.shellcheck.net/wiki/SC2155
```suggestion
        GKE_PROJECT=$(jq -r .project < cluster.info)
        GKE_CLUSTER_NAME=$(jq -r .name < cluster.info)
        GKE_ZONE=$(jq -r .zone < cluster.info)
        readonly GKE_PROJECT GKE_CLUSTER_NAME GKE_ZONE
```